### PR TITLE
add local_zone and domain_insecure parameters

### PR DESCRIPTION
**Pull request by b4ldr**
This is an update of #151 and #152

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -6,6 +6,8 @@
   Hash $forward,
   Hash $stub,
   Hash $record,
+  Optional[Unbound::Local_zone] $local_zone,
```
Comment (xaque208) [inline manifests/init.pp]: Just looking over the man page, looks like multiple local-zone definitions are allowed.  This should probably be `Optional[Array[Unbound::LocalZone]]` to allow multiple local zones.  The template will also need adjusting.

Comment (b4ldr) [reply manifests/init.pp]: `type Unbound::Local_zone = Hash[String, Unbound::Local_zone_type]` so you can pass multiple zones here, template already accounts for this

Comment (xaque208) [reply manifests/init.pp]: Yep, I just missed that.

**Comment by xaque208** [comment]
This looks good.  Thanks for the contribution.

**Pull request merged**